Removed default menus, added new font to menu, got rid of main menu u…

ConnorSheremeta opened the pull request:
…nneeded content, removed saves

ConnorSheremeta committed:
Removed default menus, added new font to menu, got rid of main menu unneeded content, removed saves

vppatel111 commented on HITDAD/js/rpg_windows.js:
```diff
@@ -1812,7 +1805,7 @@ Window_MenuStatus.prototype.setPendingIndex = function(index) {
 
 function Window_MenuActor() {
-    this.initialize.apply(this, arguments);
+    fthis.initialize.apply(this, arguments);
```
Was this intentional or is this a typo?

vppatel111 commented:
A couple more issues:

- It appears that the pressing the "Esc" key now does nothing. This is a problem because you can't back out of "Continue" menu or pause during a game.
- Additionally, you removed the "Options" option from the menu but we still want to allow the player to change the game volume from the title screen, we only wanted to remove the TMJumpAction configuration, please see here: https://sevenpointsoffailure.slack.com/archives/CSDFVG2UA/p1584633475047700

Note: You can remove menu options from the ingame menu from Database > System > Menu Commands. You only need to change the code for TMJumpAction to remove their menu item, the others can be done through the interface.

vppatel111 reviewed (changes requested):
See above comments.

ConnorSheremeta committed:
Address review

ConnorSheremeta commented:
Menus now removed properly, removed typo, esc now works, all options now on options selection again other than jap option (we want all others right?), save and continue disabled

ConnorSheremeta committed:
Merge branch 'master' into cds-change-menu

ConnorSheremeta committed:
mapinfos changes

mikegreber committed:
Maps updated

mikegreber committed:
Merge branch 'map-updates' into cds-change-menu

mikegreber committed:
Mapinfos fixed

mikegreber reviewed (approved):
Looks good, i merged my map-updates in here and did a fix on a bug that showed up with the level names not displaying properly. For some reason the menu font doesn't load in when the game is initially loaded but we can figure that out later.
